.pipelines/ci/templates/build-powertoys-steps.yml

```diff
@@ -78,15 +79,29 @@ steps:
 - task: VisualStudioTestPlatformInstaller@1
   displayName: Ensure VSTest Platform
 
+- ${{ if eq(parameters.enableCaching, 'true') }}:
+  - task: NuGetToolInstaller@1
+    displayName: Install NuGet
+
+  - script: nuget restore packages.config -SolutionDirectory .
+    displayName: 'nuget restore packages.config'
+
 - task: VSBuild@1
   displayName: 'Build PowerToys.sln'
   inputs:
     solution: '**\PowerToys.sln'
     vsVersion: 17.0
     platform: '$(BuildPlatform)'
     configuration: '$(BuildConfiguration)'
-    msbuildArgs: -restore ${{ parameters.additionalBuildArguments }}
+    ${{ if eq(parameters.enableCaching, 'true') }}:
+      msbuildArgs: -restore ${{ parameters.additionalBuildArguments }} -graph -reportfileaccesses -p:MSBuildCacheEnabled=true -p:MSBuildCacheLogDirectory=$(Build.ArtifactStagingDirectory)\logs\MSBuildCache -bl:$(Build.ArtifactStagingDirectory)\logs\PowerToys.binlog -ds:false
+    ${{ if ne(parameters.enableCaching, 'true') }}:
```

Comment:
nit: prefer `${{else}}`

Reply:
I... didn't know that was a thing. That makes it much cleaner!